frontend/src/components/activity/content/ColumnLayout.vue

```diff
@@ -12,7 +12,6 @@
                       :min-width="minWidth(slot)"
                       :max-width="maxWidth(slot)"
                       :color="color"
-                      :show-header="!isRoot"
```

Comment:
Wieso diese Änderung?

Reply:
Ich habe das gemacht weil in den Alice Fixtures nicht alle Activities ein einspaltiges ColumnLayout als Root haben, und ich mit Spaltenbreiten etc. experimentieren wollte. Inzwischen haben wir bessere Dev-Daten, daher kann ich das reverten. Merci für den Hinweis!